src/GovUk.Education.ExploreEducationStatistics.Admin/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs

```diff
@@ -343,6 +352,39 @@ private IActionResult RedirectToLoginPageWithError(string returnUrl)
             return LocalRedirect(returnUrl);
         }
 
+        public async Task<IActionResult> OnPostConfirmationAsync(string returnUrl = null)
+        {
+            // This method should no longer be used but just in case, return user to login page if it is hit.
+            var pageRedirect = RedirectToPage("./Login", new {ReturnUrl = returnUrl ?? Url.Content("~/")});
+            return await Task.FromResult(pageRedirect);
+        }
+
+        private async Task<IActionResult> HandleExpiredInvite(UserInvite inviteToSystem, string email)
+        {
+            var releaseInvites = GetUserReleaseInvites(email);
+            var publicationInvites = GetUserPublicationInvites(email);
+            _usersAndRolesDbContext.UserInvites.Remove(inviteToSystem);
+            _contentDbContext.UserReleaseInvites.RemoveRange(releaseInvites);
+            _contentDbContext.UserPublicationInvites.RemoveRange(publicationInvites);
```

Comment:
I think we might want to check the created dates of all these invites too, rather than deleting them all?